.changeset/silly-plums-buy.md

```diff
@@ -0,0 +1,9 @@
+---
+"docs": major
+"@espressive/cascara": major
+"@espressive/design-tokens": major
+"@espressive/icons": major
+"@espressive/eslint-config": major
+---
+
+Chore(FDS-359) Move eslint-plugin-json-format to peerDependencies in the eslint config
```

Comment:
@rociobaena we definitely don't want all of these packages to get a major version bump. Just the eslint config. The nice thing with Changeset files is that we can just delete this file if we do not want this to happen. We should only have the `selfish-jars-fetch.md` changeset file and remove this one.